include/bluetooth/hci.h

```diff
@@ -204,6 +204,10 @@ struct bt_hci_cmd_hdr {
 						BT_LE_FEAT_BIT_SLAVE_FEAT_REQ)
 #define BT_FEAT_LE_DLE(feat)                    BT_FEAT_TEST(feat, 0, 0, \
 						BT_LE_FEAT_BIT_DLE)
+#define BT_FEAT_LE_PHY_2M(feat)                 BT_FEAT_TEST(feat, 0, 1, \
+						BT_LE_FEAT_BIT_PHY_2M - 8)
+#define BT_FEAT_LE_PHY_COD(feat)                BT_FEAT_TEST(feat, 0, 1, \
+						BT_LE_FEAT_BIT_PHY_COD - 8)
```

Comment:
Another reason why a separate macro makes sense is that LE doesn't seem to have a concept of feature pages (which is also a required input to the original macro).